src/pages/programs/program-observation-submission/program-observation-submission.ts

```diff
@@ -97,12 +97,10 @@ export class ProgramObservationSubmissionPage {
       .then((data) => {
```

Comment:
This page is changed every where for submissionId and submissionStatus to _id and status